packages/video_player/video_player_avfoundation/pigeons/messages.dart

```diff
@@ -54,4 +96,16 @@ abstract class AVFoundationVideoPlayerApi {
   void pause(int textureId);
   @ObjCSelector('setMixWithOthers:')
   void setMixWithOthers(bool mixWithOthers);
+  @ObjCSelector('isPictureInPictureSupported')
+  bool isPictureInPictureSupported();
+  @ObjCSelector('setPictureInPictureOverlaySettings:')
+  void setPictureInPictureOverlaySettings(
+      SetPictureInPictureOverlaySettingsMessage msg);
```

Comment:
New code should not use the message object pattern (and in fact the existing code has since been updated not to use it).